### PR TITLE
Update Traction-Digital.com.xml, remove unused rules

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Traction-Digital.com.xml [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
 
-	<target host="*.traction-digital.com" />
-
-
-	<!--	Tracking cookies:
-					-->
-	<securecookie host="^\.traction-digital\.com$" name="^__utm\w+$" />
-
+	<target host="traction-digital.com" />
+	<target host="www.traction-digital.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Traction-Digital.com.xml]: `info.traction-digital.com`

File: src/chrome/content/rules/Traction-Digital.com.xml [hunk 2]
```diff
@@ -1,33 +1,14 @@
...
-
-		- dujdlx1yovuet.cloudfront.net
-
-			- cdn
-
-
-	(www.): plaintext reply
-
 -->
 <ruleset name="Traction-Digital.com (partial)">
```
Comment (J0WI) [inline src/chrome/content/rules/Traction-Digital.com.xml]: This is no longer partial.

**Comment by cschanaj** [comment]
@J0WI done and Travis is finally passing. thanks!!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**